packages/now-cli/src/commands/login.js

```diff
@@ -255,10 +258,9 @@ const login = async ctx => {
 
   output.debug(`Saved credentials in "${hp(getNowDir())}"`);
 
-  console.log(
-    `${chalk.cyan('> Congratulations!')} ` +
-    `You are now logged in. In order to deploy something, run ${cmd('now')}.`
-  );
+  console.log(`${chalk.cyan('Congratulations!')} ` + `You are now logged in`);
```

Comment:
Why print this message? Shouldn't the `doc.print()` contain the congratulations part too?

Reply:
Hmm, I'd leave it like this to close the "login" process, before we print a guide of how to use `now`